client/my-sites/plugins/main.jsx

```diff
@@ -196,7 +184,7 @@ const PluginsMain = React.createClass( {
 
 		if ( selectedSite ) {
 			emptyContentData.actionURL = '/plugins/' + selectedSite.slug;
-			if ( selectedSite.jetpack ) {
+			if ( this.props.isJetpackSite( selectedSite.ID ) ) {
```

Comment:
Ah, good point. I've addressed that for `canJetpackSiteManage` and `canJetpackSiteUpdateFiles`, as they also have cases where they are called for the selected site.